### PR TITLE
DB diagram changed. Package update to cover default rc-util install n…

### PR DESCRIPTION
**Pull request by UlysseFG**
…ot installing the most recent version. Moving env to base directory.

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -130,7 +131,7 @@ coverage run ./manage.py test
 
 ## Database diagram
 
-![Database Schema Diagram](docs/fms_database_diagram.png)
+[Database Schema Diagram](https://dbdiagram.io/d/604aa285fcdcb6230b23c0e7)
```
Comment (jleblond) [inline backend/README.md:134]: If this is on master, maybe it should include the latest tables, such as Sample Kind, Sample Lineage, Protocol, etc. for example?

Comment (UlysseFG) [reply backend/README.md:134]: yeah you are right i must be sleep walking to do that on master with the old diagram ... I'll include the new changes

Comment (UlysseFG) [reply backend/README.md:134]: Ok updated the diagram to be in sync with master instead ...

**Review by jleblond — APPROVED** [review]

File: backend/README.md [hunk 2]
```diff
@@ -23,13 +23,14 @@ Python package dependencies are listed in `requirements.txt`.
...
+  1. Set up a virtual environment with Python 3.8 or later, and install 
      dependencies into it:
      
      ```bash
-     virtualenv -p python3 ./env
+     cd .. # move into base repository directory
+     virtualenv -p python3.8 ./env
      source env/bin/activate
-     pip install -r requirements.txt
+     pip3.8 install -r ./backend/requirements.txt
```
[the remark this answers is not in the document]
Comment (UlysseFG) [reply backend/README.md:33]: This is mainly to stay in sync with the deployment i've done on de DEV server and that will eventually be extended to QC. The /instance/ directory with the uwsgi config and auto_deploy was moved to the repo root. The uwsgi version is dependent on the python version so i moved the env along to cover it.

**Pull request merged**